src/server/game/DataStores/DBCEnums.h

```diff
@@ -310,8 +310,8 @@ enum MapFlags
 
 enum AbilytyLearnType
```

Comment:
Ability

Reply:
It's fine

Reply:
Hmm yeah, @Ali-Khazaee you could fix it with another PR after this one has been merged because it looks ugly :P